### ISSUE TITLE
The data was sent to Grafana because it was delayed by 1 minute.

### ISSUE THREAD

**Issue by SIKIM92** [influxdata/telegraf#10134]
### Relevent telegraf.conf

```toml
i used Cloudwatch plugin.
The data was sent to Grafana because it was delayed by 1 minute.

i used Cloudwatch plugin.
namespaces = ["AWS/ELB"]  In the case of , there is no problem.

There is no delay by sending data to Grafana every minute without using the delay = "1m" option.

=====code======
[[inputs.cloudwatch]]
  region = "ap-northeast-2"
  period = "1m"
  interval = "1m"
  namespaces = ["AWS/ELB"]

  [[inputs.cloudwatch.metrics]]
     names = ["Latency", "RequestCount","UnHealthyHostCount","HTTPCode_Backend_4XX","SurgeQueueLength","HTTPCode_Backend_5XX"]

    [[inputs.cloudwatch.metrics.dimensions]]
      name = "LoadBalancerName"
      value = "*"

But when you use Application ELB, delay = "1m" 
If you do not give options, you will not be able to send data every minute.

=======code========
[[inputs.cloudwatch]]
region = "ap-northeast-2"
period = "1m"
interval = "1m"
delay = "1m"

namespaces = ["AWS/ApplicationELB"]

[[inputs.cloudwatch.metrics]]
names = ["RequestCount", "HTTPCode_Target_2XX_Count", "HTTPCode_Target_4XX_Count", "HTTPCode_Target_5XX_Count", "HTTPCode_ELB_4XX_Count", "HTTPCode_ELB_5XX_Count"]

in conclusion
When using namespaces = ["AWS/ApplicationELB"]
Do I have to use the delay = "1m" option?

I want to send data to Grafana immediately without using the delay = "1m" option.
If delay = "1m" is used, a 1-minute delay occurs.


2. namespaces = ["AWS/ELB"]
In the case of , data is immediately transmitted to Grafana every minute without using delay = "1m".

Is there a difference between namespaces = ["AWS/ELB"] and namespaces = ["AWS/ApplicationELB"?

I just want to send data to Grafana immediately without using the delay = "1m" option while using namespaces = ["AWS/ApplicationELB"]
```

### System info

telegraf 1.20

### Steps to reproduce

1.
2.
3.
...

### Expected behavior

idt

### Actual behavior

no
